pkg/util/certificates/csr.go

```diff
@@ -93,7 +93,7 @@ func NewCertificateRequest(privateKey interface{}, subject *pkix.Name, dnsSANs [
 	switch privateKey := privateKey.(type) {
 	case *ecdsa.PrivateKey:
 		switch privateKey.Curve {
-		case elliptic.P224(), elliptic.P256():
```

Comment:
On Fedora ``P.224`` is removed intentionally [1]. Does not have to apply for other distributions.

[1] https://bugzilla.redhat.com/show_bug.cgi?id=1038683